GCRCatalogs/catalog_configs/dc2_object_run1.2p_tract4850.yaml

```diff
@@ -1,7 +1,7 @@
 subclass_name: dc2_object.DC2ObjectCatalog
```

Comment:
Use the `alias` feature instead of repeating the content of the yaml file.

Suggestion: change this whole config to just:
```yaml
alias: dc2_object_run1.2p_v4_tract4850
included_by_default: true
```